packages/@aws-cdk/aws-autoscaling/package.json

```diff
@@ -122,6 +122,7 @@
       "export:@aws-cdk/aws-autoscaling.IAutoScalingGroup",
       "props-physical-name:@aws-cdk/aws-autoscaling.AutoScalingGroupProps",
       "props-physical-name:@aws-cdk/aws-autoscaling.ScheduledActionProps",
+      "props-physical-name:@aws-cdk/aws-autoscaling.WarmPoolProps",
```

Comment:
Please write a doc for this instead of excluding.

Reply:
Will write a doc.

Reply:
I think we need to keep it. The following is an error message when I remove this line:

```
error: [awslint:props-physical-name:@aws-cdk/aws-autoscaling.WarmPoolProps] Every Resource must have a single physical name construction property, with a name that is an ending substring of <cfnResource>Name
```

An user cannot assign a name to an warm pool unlike other resources, i.e. lifecycle hooks.

Reply:
Good call. Keep it as is.